app/src/main/java/com/example/campusguide/MapsActivity.kt

```diff
@@ -60,6 +62,24 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
         val hall = LatLng(45.497290, -73.578824)
         mMap.addMarker(MarkerOptions().position(hall).title("Hall Building"))
         mMap.animateCamera(CameraUpdateFactory.newLatLngZoom(hall, 17.0f))
+
+        // Update switch campus button listener
+        val switchCampusToggle: ToggleButton = findViewById(R.id.switchCampusButton)
```

Comment:
Should we extract the campus toggle code into its own class? Otherwise this class is going to end up with too many responsibilities.

Reply:
yeah we should. i'll try to figure out how to do that tonight but no promises it'll be done for tomorrow